velox/common/base/Exceptions.h

```diff
@@ -16,12 +16,23 @@
 
 #pragma once
 
+#ifdef ARROW_USE_GLOG
```

Comment:
Seems like unrelated change. Please, remove.

Reply:
will remove it.